src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -586,12 +658,17 @@ export default {
 
 		this.itemsInBasket = _map(_get(baseData, 'shop.basket.items.values'), 'id');
 
+		console.log('without', _without(this.realCategoryIds, this.customCategoryIds));
+		// RETURNING: without [60, 103, 95, 56, 65, 52, 68, 70, 67, 98, 11, 76]
+		console.log('Take and without', _take(_without(this.realCategoryIds, this.customCategoryIds), ssrRowLimiter));
+		// RETURNING: Take and without [60, 103]
 		// Read the SSR ready loan channels from the cache
 		try {
 			const categoryData = this.apollo.readQuery({
 				query: loanChannelQuery,
 				variables: {
-					ids: _take(this.realCategoryIds, ssrRowLimiter),
+					// ERROR IS BEING THROWN HERE or withing realCategoryIds or customCategoryIds
+					ids: _take(_without(this.realCategoryIds, this.customCategoryIds), ssrRowLimiter),
```

Comment:
To me it looks like the error looks to be thrown here, while we're joining these two arrays